fabfile/setup.py

```diff
@@ -103,7 +106,7 @@ def redhat_install_packages():
 
     with hide('stdout'):
         sudo('yum install -q -y libxml2-devel libxslt-devel'
-            ' openssl-devel pam-devel readline-devel libcurl-devel git')
+            ' openssl-devel pam-devel readline-devel libcurl-devel git libuuid-devel')
```

Comment:
is `libuuid-devel` for the extension?

Reply:
Yes, it is. Now I will parameterize package names to be installed and will add libuuid-devel when the case is valgrind tests

Reply:
yep and removed it as we do not require it anymore